* allow test scripts to inherit environment
* set stack size ulimit to unlimited
* slightly raise tolerance in some tests
author: Miguel Dias Costa (National University of Singapore)
--- BerkeleyGW-4.0/testsuite/run_testsuite.sh.orig	2024-03-20 01:10:39.000000000 +0800
+++ BerkeleyGW-4.0/testsuite/run_testsuite.sh		2024-03-20 11:11:19.179296008 +0800
@@ -1,4 +1,4 @@
-#!/bin/bash -l
+#!/bin/bash
 #
 # Copyright (C) 2005-2009 Heiko Appel, David Strubbe
 #
@@ -66,6 +66,8 @@
 # Run all tests. Takes as first argument a list of testfile names.
 function run_testsuite() {
 
+ulimit -s unlimited
+    
 tests="$1"
 
 # Check for 'preserve working directories' flag.
--- BerkeleyGW-4.0/MeanField/Utilities/mf_convert_wrapper.sh.orig	2024-03-20 11:11:41.219446558 +0800
+++ BerkeleyGW-4.0/MeanField/Utilities/mf_convert_wrapper.sh		2024-03-20 11:11:56.019547652 +0800
@@ -1,4 +1,4 @@
-#!/bin/bash -l
+#!/bin/bash
 
 # David Strubbe, October 2010
 # Wrapper for mf_convert.x
--- BerkeleyGW-4.0/testsuite/GaAs-EPM/GaAs.test.orig	2024-03-20 11:55:47.333669371 +0800
+++ BerkeleyGW-4.0/testsuite/GaAs-EPM/GaAs.test	2024-03-20 11:57:15.045252252 +0800
@@ -32,7 +32,7 @@
 Output : WFN.out
 Input  : WFN.in PIPE
 
-Precision : 8e-15
+Precision : 8e-14
 match ; Eigenvalue  1 at k-pt 1   ; GREP(WFN.out,   "kpoint          1", 2,  1); -0.2710614199849328
 match ; Eigenvalue 10 at k-pt 1   ; GREP(WFN.out,   "kpoint          1", 2, 10);  1.2565373697755460
 match ; Eigenvalue 18 at k-pt 2   ; GREP(WFN.out,   "kpoint          2", 2, 18);  2.1322637363008994
